src/feature/stats/rephist.c

```diff
@@ -1527,6 +1533,52 @@ rep_hist_load_state(or_state_t *state, char **err)
   return 0;
 }
 
+/** Collects BandwidthStatistics parameter from consensus.
+ *  Returns 0 iff BandwidthStatistics is 0 or
+ *  the consensus param is 0. */
```

Comment:
This description is confusing. It doesn't mention what happens when the option is auto or 1. (And how that is different.)

What happens when the BandwidthStatistics option is 1, but the BandwidthStatistics consensus parameter is 0 ?

Try to rewrite the comment based on other consensus parameter function comments.

Reply:
I think when it's value is explicitly set to 1 then we don't check the consensus parameter.
[citation](https://gitweb.torproject.org/torspec.git/tree/proposals/313-relay-ipv6-stats.txt#n305).